src/Java.Interop.Tools.JavaTypeSystem/JavaModels/JavaConstructorModel.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace Java.Interop.Tools.JavaTypeSystem.Models
+{
+	public class JavaConstructorModel : JavaMethodModel
+	{
+		public JavaConstructorModel (string javaName, string javaVisibility, bool javaAbstract, bool javaFinal, bool javaStatic, JavaTypeModel javaParentType, string deprecated, string jniSignature, bool isSynthetic, bool isBridge)
```

Comment:
`javaAbstract` doesn't make sense for `JavaConstructorModel`; how is a constructor supposed to be `abstract`?  Or `final` for that matter?

What is meant by `javaParentType`? The declaring type?